cmake/modules/AddSwift.cmake

```diff
@@ -981,6 +981,9 @@ function(_add_swift_library_single target name)
      "${SWIFT_SDK_${SWIFTLIB_SINGLE_SDK}_OBJECT_FORMAT}" STREQUAL "ELF")
     list(APPEND link_flags "-fuse-ld=gold")
   endif()
+  if("${SWIFTLIB_SINGLE_SDK}" STREQUAL "FREEBSD")
+    list(APPEND link_flags "-fuse-ld=gold")
```

Comment:
Can I ask why this isn't covered by the previous condition? I can see SWIFT_ENABLE_GOLD_LINKER defaulting to on on FreeBSD, but does it need to be forced on?